metricbeat/docs/aws-credentials-examples.asciidoc

```diff
@@ -25,7 +26,6 @@ metricbeat.modules:
 ----
 
 * Use shared AWS credentials file
-
```

Comment:
Not sure why you removed the blank line. If you want the example to be indented below the bullet item (it should be), then use a plus (+).  So you would have:

```
* Use shared AWS credentials file
+ 
[source,yaml]
```
If you change the indentation, please do that throughout the topic. I have a to-do item to edit this content to fix the indentation and other stuff, but it might be awhile before I can do that. 

If you need to indent multiple paragraphs, put a plus between all the paragraphs that need to be indented.